src/libs/actions/IOU.js

```diff
@@ -291,7 +299,90 @@ function payIOUReport({
             }
             Onyx.merge(ONYXKEYS.IOU, {error: true});
         })
-        .finally(() => Onyx.merge(ONYXKEYS.IOU, {loading: false})),
+        .finally(() => {
+            Onyx.merge(ONYXKEYS.IOU, {loading: false});
+        }),
+    url);
+}
+
+/**
+ * Sends money by creating and paying for an IOU Report and then retrieves the iou and chat reports to trigger updates to the UI.
```

Comment:
NAB: For consistency.
```suggestion
 * Sends money by creating and paying for an IOU Report and then retrieves the IOU and chat reports to trigger updates to the UI.
```